### ISSUE TITLE
deviceClient.StartApp("com.android.app"); doesn't work

### ISSUE THREAD

**Issue by evolution1987** [SharpAdb/AdvancedSharpAdbClient#90]
### Describe the bug

HI! I have updated package from 2.5.8 to 3.0.9 buy found that deviceClient.StartApp("com.android.app"); doesn't open apk on smartphone There are no errors in log. I use NET Framework 4.8

### Steps to reproduce the bug

1. Install facebook apk
2. deviceClient.StartApp("com.facebook.katana");

### NuGet package version

3.0.9

### .NET Platform

.NET Framework 4.8.x

### Platform type

Windows

### System version

windows 10 pro

### IDE

Visual Studio 2022

**Comment by wherewhere** [comment]
It works on my device...
![Screenshot_20240208-115529_Termux.png](https://github.com/SharpAdb/AdvancedSharpAdbClient/assets/27689196/cc049c85-fdb3-480f-845e-0fd20661b4b6)

**Comment by wherewhere** [comment]
Here is the screenrecord...

https://github.com/SharpAdb/AdvancedSharpAdbClient/assets/27689196/bf3e1a35-db1d-4db1-a2d8-4fc8c9ebfcdb

**Comment by evolution1987** [comment]
On my visual studio Start app works on 2.5.8 version but not in 3.0.9

**Comment by wherewhere** [comment]
.net4.8 also works.
Try using `client.StartApp(device, "com.bilibili.app.in")`

![Screenshot_20240208-195941_Termux.png](https://github.com/SharpAdb/AdvancedSharpAdbClient/assets/27689196/1c5d6ab0-f783-40dc-bb86-40e5acba4c8c)

**Comment by wherewhere** [comment]
So what's your old code?

**Comment by evolution1987** [comment]
This code with 3.0.9 version and StartApp doesn't work

**Comment by wherewhere** [comment]
But it works on my device. I need to know the difference to figure out what happened...
And why you connect a connected device?

**Comment by wherewhere** [comment]
```cs
var serial = client.GetDevices().FirstOrDefault().Serial; //You get the first connected device's serial.
client.Connect(serial); //Then you connect it again.
var deviceData = client.GetDevices().FirstOrDefault(); //Now I don't know which device you would get...
```

**Comment by evolution1987** [comment]
You are right but in any case I can't start apk after installing, comman doesn't run application, nothing happens. But all works on 2.5.8 So I think there is some promler in latest version of Advanced Sharp Adb I try to found whre is problem Can you show actual examples on real smartphone or maybe you need test on facebook apk?

**Comment by evolution1987** [comment]
Tell me please, in method client.Connect("ip:port") can I use name of device as ip:port?

**Comment by wherewhere** [comment]
No, you can only use ip and port like `192.168.1.19:37146`.

**Comment by evolution1987** [comment]
So, if I coonnected my smartphone via USB I can ignore the line **client.Connect("ip:port")**?

**Comment by wherewhere** [comment]
Yes.

**Comment by evolution1987** [comment]
Thank you very much, I will try to discover my problem again

**Comment by evolution1987** [comment]
Hello! I can run app usimg shell as you said me, but comand StartApp from documentaion doesn't work for me

**Comment by tinngv** [comment]
Same issue, 2.5.8 runs well, but 3.0.9+ cannot start the app event if the shell command runs ok. Now I still use 2.5.8. :(

**Comment by tinngv** [comment]
Change code to await device.StartAppAsync("packagename") it works fine :)

**Comment by wherewhere** [comment]
I have no idea...

**Comment by regregoff** [comment]
I have the same problem. I also could not launch the application using 
```C#
AdbClient client = new AdbClient();
DeviceData device = client.GetDevices().FirstOrDefault();
deviceClient.AdbClient.ExecuteShellCommand(device, "monkey -p org.telegram.messenger.web 1"); 
```
but the application was launched with the command 
```C#
AdbClient client = new AdbClient();
DeviceData device = client.GetDevices().FirstOrDefault();
ConsoleOutputReceiver receiver = new ConsoleOutputReceiver();
client.ExecuteRemoteCommand("monkey -p org.telegram.messenger.web 1", device, receiver);
```
I'm a complete novice in C#, but if I understand correctly, the problem is that when calling ExecuteRemoteCommand, the receiver is not transmitted, and without it the commands are not executed

**Comment by liditingziya** [comment]
I encountered the same problem.
![image](https://github.com/user-attachments/assets/f971ff1e-d380-4ad4-b8be-6aff80ab5d72)
This code works in `v2.5.8`, but it has no effect in versions above it.
